modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -867,6 +869,8 @@ private CompletableFuture<Void> startPartitionAndStartClient(
                     }
                 }), ioExecutor)
                 .thenAcceptAsync(updatedRaftGroupService -> inBusyLock(busyLock, () -> {
+                    updatedRaftGroupService.updateConfiguration(configurationFromAssignments(union(newPartAssignment, oldPartAssignment)));
```

Comment:
So, you claim that we will finish here both in case of node recovery on restart and common pending assignments update handling?

Reply:
I've changed nothing in this code regarding recovery. As far as I can see from the code, for the persistence storage `shouldStartGroupFut` will return true, so the flow will be the same, as for common pending assignments update handling

Reply:
I mean that we should union pending and stable both for recovery and common events processing. Please check whether it's true.

Reply:
I've checked, we start pending and stable clients for recovery as well